projects/WithRuleset/WithRuleset.csproj

```diff
@@ -7,6 +7,7 @@
   
   <ItemGroup>
     <Compile Include="../common/Code.cs" />
+    <PackageReference Include="Microsoft.DotNet.PlatformAbstractions" Version="1.1.1" />
```

Comment:
Why added here? This file does not seem to be referenced by any of the test cases